events/class_models/event_models.py

```diff
@@ -1,7 +1,61 @@
 from django.db import models
+from django.db.models import Count
 from django.core.exceptions import ValidationError
 
 
+class EventQuerySet(models.QuerySet):
+    def get_all_meetings(self):
```

Comment:
INMO we should have alaso have a query of all events (single participant)

Reply:
done. thanks 😄